enterprise/suppress-user/internal/badgerdb/badgerdb.go

```diff
@@ -239,6 +242,15 @@ func (b *Repository) start() error {
 			if err == nil {
 				goto again
 			}
+			lsmSize, vlogSize, totSize, err := misc.GetBadgerDbUsage(b.db.Opts().Dir)
+			if err != nil {
+				b.log.Errorf("Error while getting badgerDB usage: %v", err)
+				continue
+			}
+			statName := "suppress-user"
+			b.stats.NewTaggedStat("badger_db_size", stats.GaugeType, stats.Tags{"name": statName, "type": "lsm"}).Gauge((lsmSize))
+			b.stats.NewTaggedStat("badger_db_size", stats.GaugeType, stats.Tags{"name": statName, "type": "vlog"}).Gauge((vlogSize))
+			b.stats.NewTaggedStat("badger_db_size", stats.GaugeType, stats.Tags{"name": statName, "type": "total"}).Gauge((totSize))
```

Comment:
we don't need necessarily the `type=total`, since we can synthesis it by comping the other metrics during query time.
```suggestion
```